src/map/clif.c

```diff
@@ -10004,6 +10004,23 @@ static bool clif_process_message(struct map_session_data* sd, bool whisperFormat
 	return true;
 }
 
+/* clif_pk_mode_message
+ * Displays a message if the player joins a PK Zone
+ * (Only enabled in pk_mode)
+ * @param sd
+ * @return void
+ *------------------------------------------*/
+inline void clif_pk_mode_message(struct map_session_data * sd)
+{
+	if (battle_config.pk_mode && battle_config.pk_mode_mes &&
+	    sd && map[sd->bl.m].flag.pvp) {
+		// 407: You've entered the PK Zone.
+		clif_disp_overhead_(&sd->bl, msg_txt(sd,407), BC_SELF);
```

Comment:
I'd like to use a different display mode, because this one spams the chat window. Is there a display_overhead option which will not be displayed in the chat window?

Reply:
How come im getting error on this part? `clif_disp_overhead_(&sd->bl, msg_txt(sd,407), BC_SELF);`

Reply:
Can you specify the error?

Reply:
```
clif.c:10150:3: warning: implicit declaration of function ‘clif_disp_overhead_’ [-Wimplicit-function-declaration]
   clif_disp_overhead_(&sd->bl, msg_txt(sd,500), BC_SELF);
```

and

```
/tmp/ccCDv4ka.ltrans18.ltrans.o: In function `clif_parse_LoadEndAck.part.76.ltopriv.592':
<artificial>:(.text+0xb13b): undefined reference to `clif_disp_overhead_'
collect2: error: ld returned 1 exit status
Makefile:69: recipe for target 'map-server' failed
make[1]: *** [map-server] Error 1
make[1]: Leaving directory '/home/wickedro/wickedz/src/map'
Makefile:51: recipe for target 'map' failed
make: *** [map] Error 2
```

Reply:
Shouldn't be a implicit declaration. The definition (https://github.com/rathena/rathena/blob/feature/pk_mode_mes/src/map/clif.c#L9566) is in the same file some lines above the call (https://github.com/rathena/rathena/blob/feature/pk_mode_mes/src/map/clif.c#L10018). Also the declaration is in the header (https://github.com/rathena/rathena/blob/feature/pk_mode_mes/src/map/clif.h#L834) Are you sure you merged it correctly?

Reply:
I'm sorry I already tested it. It' definitely an issue on your side. Also I'm using `clif_disp_overhead_` not `clif_disp_overhead` and your call is in a different line (`clif.c:10150`), pretty sure you're not testing on this branch. Please try to checkout this clean branch. If that works you should sanitize your merge.